src/views/visyn/interfaces.ts

```diff
@@ -168,6 +168,17 @@ export type VisynDataViewPluginType<
      * TODO:: Type to IReprovisynServerColumn when we merge that into tdp_core
      */
     dataDesc: IServerColumn[] | any[];
+
+    /**
+     * List of items which are filtered out of the view. Ids match the idtype from 'desc.idtype'
+     */
+    idFilter: string[];
```

Comment:
For me the property name was a bit confusing. I would suggest the following renaming. What do you think?

```suggestion
    /**
     * List of items which are filtered out in this view. The ids must match the idtype from 'desc.idtype'
     */
    filteredOutIds: string[];
```